CPT331.WebAPI/Api/TestController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+
+namespace CPT331.WebAPI.Api
+{
+    public class TestController : ApiController
```

Comment:
Excellent, let's do that:
 - All UI stuff to be removed from CPT331.WebAPI
 - A web front end to be created - CPT331.Web perhaps?

Does that look OK to you?